app/code/UAE/GroupedProduct/view/frontend/web/css/source/_module.less

```diff
@@ -18,6 +18,54 @@
       }
     }
   }
+
+  .bottom-bar {
+    position: fixed;
+    bottom: 0;
+    width: 100%;
+    background-color: gainsboro;
+    min-height: 50px;
+    z-index: 9999;
+
+    & > .back_to_top {
+      float: left;
+      padding: 30px 0 0 20px;
+      display: flex;
+      align-items: center;
+      cursor: pointer;
+
+      & > img {
+        width: 15px;
+        height: 15px;
+      }
```

Comment:
@Andrewb89 - One more point of FE feedback: The arrow icons should be implemented with the icon font, like Magento uses by default. This is an advanced CSS technique and is a little tricky, but it can be done in Magento by calling the .lib-icon-font() mixin with certain parameters. The advantages of this are that we don't have to include the arrrow.svg file or add the <img> elements in the template, and we can also change the color of the icon (since it all comes from CSS and fonts).
(Once again, I already did this in a later commit, so you don't need to change anything this time. I just wanted you to know for next time.)